mteb/tasks/Classification/tel/TeluguAndhraJyotiNewsClassification.py

```diff
@@ -0,0 +1,38 @@
+from __future__ import annotations
+
+from mteb.abstasks import AbsTaskClassification
+from mteb.abstasks.TaskMetadata import TaskMetadata
+
+
+class TeluguAndhraJyotiNewsClassification(AbsTaskClassification):
+    metadata = TaskMetadata(
+        name="TeluguAndhraJyotiNewsClassification",
+        description="A Telugu dataset for 5-class classification of Telugu news articles",
+        reference="https://github.com/AnushaMotamarri/Telugu-Newspaper-Article-Dataset",
+        dataset={
+            "path": "mlexplorer008/telugu_news_classification",
+            "revision": "3821aa93aa461c9263071e0897234e8d775ad616",
+        },
+        type="Classification",
+        category="s2s",
+        date=("2014-01-01", "2018-01-01"),
+        eval_splits=["test"],
+        eval_langs=["tel-Telu"],
+        main_score="f1",
+        form=["written"],
+        domains=["News"],
+        task_subtypes=["Topic classification"],
+        license="MIT",
+        socioeconomic_status="mixed",
+        annotations_creators="derived",
+        dialect=[],
+        text_creation="found",
+        bibtex_citation=None,
```

Comment:
Is there no citation? Then I would do "" (it is still being debated, but @isaac-chung I assume you agree, but otherwise let me know)

Reply:
Yep I agree.